styles/main.css

```diff
@@ -211,3 +211,115 @@ nav {
   margin: 10px;
   border-radius: 10px;
 }
+
+/* Contact me section */
+.contact-me {
+  min-height: 779px;
+  min-width: 375px;
+  width: 100%;
+  background: url(../images/contact-form-shapes-mobile.png) 100% 45px no-repeat;
+  background-size: auto 709px;
+  background-color: #6070ff;
+  border-top-left-radius: 100px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: -160px;
+}
+
+.contact-header {
+  margin-top: 90px;
+  text-align: center;
+}
+
+.contact-header h1 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+.contact-header p {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  margin-top: -12px;
+  padding: 10px;
+}
+
+form {
+  padding: 30px 24px 0 24px;
+}
+
+form ul li:not(:last-child) {
+  margin-bottom: 20px;
+}
+
+form ul li input {
+  width: 90%;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  background: white;
+  font-size: 15px;
+  line-height: 20px;
+  color: #172b4d;
+  padding: 15px 16px;
+  outline: none;
+}
+
+form ul li input:focus {
+  border: 2px solid #abc5eb;
+}
+
+form ul li textarea {
+  background: white;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 15px 0;
+  outline: none;
+  width: 93%;
+  height: 140px;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  color: #b3bac5;
+  font-size: 15px;
+}
```

Comment:
- [ ] [OPTIONAL] Hi team. you have done a great implementation, however, I see that the background color for your text area is `white` while the color itself is grey `#b3bac5`, this doesn't make the text input really accessible. Please change it to black or the color you gave the email and the name input.

![Screenshot from 2022-01-19 15-26-40](https://user-images.githubusercontent.com/58692590/150150645-5a6b2b51-9a4c-47d4-9abb-1ea9df4c12cf.png)

Reply:
Thank you for your attention